.eslintrc

```diff
@@ -1,12 +1,49 @@
 {
-  "extends": "airbnb",
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:jsx-a11y/strict",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended",
+    "prettier/react"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "tsconfig.json",
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
   "env": {
     "browser": true,
     "node": true,
     "es6": true,
     "jest": true
   },
   "rules": {
-    "global-require": off
+    "import/extensions": 0,
+    "global-require": 0,
```

Comment:
Did all of this have to be required to include the design system?

Reply:
No. I just updated eslint and other packages.